bot.js

```diff
@@ -335,36 +343,82 @@ bot.isChannel = function(name) {
 };
 
 
-bot.getReply = function(chan) {
-  return function(args) {
+bot.getReply = function(to, isPm, pmTarget) {
+  var spamReply = function(args) {
     var repStr = bot.stringifyArgs.apply(this, arguments);
-    bot.client.say(chan, repStr);
+    bot.client.say(to, repStr);
   };
-};
 
-bot.getNoticeReply = function(to) {
-  return function(args) {
-    var repStr = bot.stringifyArgs.apply(this, arguments);
+  // custom reply takes options of the following:
+  // {
+  //   lines: 2, // number of lines to spam
+  //   replaceNewLines: false, // whether to replace newlines with | while figuring out if it'll spam
+  //   pmExtra: false, // whether to PM the whole message if it overflows
+  // }
+  var customReply = function(opts, args) {
+    var maxLineChars = 460; // convenient lie; the true number is complicated due to having to account for nick-length etc
```

Comment:
That's a convenient lie too I think, but I'll grab that value and call it good enough